mis_builder/static/src/components/mis_report_widget.css

```diff
@@ -1,3 +1,19 @@
+.oe_mis_builder_content .o_list_renderer {
+    width: 100%;
+    overflow: auto;
+    position: relative;
+    /* limit height in order for the vertical scroll to work */
+    height: 80vh;
```

Comment:
I think that it would be better to use:
```suggestion
    max-height: 80vh;
```

It removes this area when having not a lot of lines and allows to vertically scroll when having multiple lines
![image](https://github.com/user-attachments/assets/0f432d66-b697-4b96-8a5a-c2a546719cca)

Reply:
yes ! Thx ! I just commited the suggested change !